createRelease.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+GREEN='\033[0;32m'
+RESET='\033[0m' # No Color
+SEPARATOR="##########################################################"
+
+function help {
+    echo ""
+    echo "USAGE: createRelease [options] [directory]"
+    echo ""
+    echo "-v  Set the version number of the release. Defaults to SNAPSHOT."
+    echo "-?  Print this help message."
+}
+
+while getopts "v:?" opt; do
+  case $opt in
+    v)
+        echo "SETTING VERSION TO $OPTARG"
+        VERSION=$OPTARG
+        ;;
+    ?)
+        help
+        exit 0
+        ;;
+    \?)
+        echo "Invalid option: -$opt" >&2
+        help
+        ;;
+  esac
+done
+
+shift $(($OPTIND - 1))
+
+if [ -z ${1+x} ]; then
+    echo "createRelease: Missing Parameter" >&2
+    help
+    exit 1
+fi
+
+STARTING_DIR=`pwd`
+REPO=$1
+REPO_NAME=`basename $REPO`
+
+if [ -a $REPO/course_management.conf ]; then
+    source $REPO/course_management.conf
+fi
+
+VERSION=${VERSION:-SNAPSHOT}
```

Comment:
I am not 100% sure if this is the best way or not. The way the script is set up right now, you can provide a version using the -v flag. If however you do not provide a version, then it will name it "SNAPSHOT".

It may be preferable to require a version always.

Thoughts?